plainly-plugin/src/ui/hooks/api/useAppmixerApi.ts

```diff
@@ -0,0 +1,16 @@
+import { get } from '@src/node/request';
+import { useMutation } from '@tanstack/react-query';
+
+export const useAppmixerUserProfile = () => {
```

Comment:
can we call this `useProfileApi` and `useUserProfile` please, has nothing to do with appmixer and we will anyway introduce another endpoint anyway